packages/orchestrator-ui-components/src/components/WfoForms/formFields/SelectField/SelectField.tsx

```diff
@@ -98,7 +98,9 @@ export function UnconnectedSelectField({
         useWithOrchestratorTheme(getSelectFieldStyles);
 
     useEffect(() => {
-        onChange(selectedValue?.value);
+        if (selectedValue && selectedValue.value !== 'undefined') {
+            onChange(selectedValue?.value);
+        }
     }, []);
```

Comment:
Fix the linting error
```suggestion
    }, [onChange, selectedValue]);
```

Reply:
Do verify if the app still works when adding this. It might lead to a render loop. I tried to fix similar cases by applying a useRef of those dependencies that causes the render loop